addons/web_editor/static/lib/odoo-editor/src/commands/deleteBackward.js

```diff
@@ -113,8 +114,8 @@ HTMLElement.prototype.oDeleteBackward = function (offset, alreadyMoved = false,
             throw UNREMOVABLE_ROLLBACK_CODE;
         }
         const parentEl = this.parentNode;
-
-        if (!isBlock(this) || isVisibleEmpty(this)) {
+        const closestLi = closestElement(this, 'li');
+        if ((closestLi && !closestLi.previousElementSibling) || !isBlock(this) || isVisibleEmpty(this)) {
```

Comment:
I don't understand why the condition is required here ? The test should still pass if you remove it because H1 is a block. What am I missing ?